project/HmrcBuild.scala

```diff
@@ -47,10 +48,16 @@ object HmrcBuild extends Build {
         Resolver.bintrayRepo("hmrc", "releases"),
         "typesafe-releases" at "http://repo.typesafe.com/typesafe/releases/"
       ),
-      crossScalaVersions := Seq("2.11.7")
+      crossScalaVersions := Seq("2.11.7"),
+      testGrouping in sbt.Test := oneForkedJvmPerTest((definedTests in sbt.Test).value)
     )
     .settings(TwirlKeys.templateImports ++= Seq("play.api.mvc._", "play.api.data._", "play.api.i18n._", "play.api.templates.PlayMagic._"))
     .settings(unmanagedSourceDirectories in sbt.Compile += baseDirectory.value / "src/main/twirl")
+
+  private def oneForkedJvmPerTest(tests: Seq[TestDefinition]) =
```

Comment:
It is not an unrelated change. I have added unit tests to test the conditional presence of optimizely snippet. For that I need to create a FakeApplication and Play's singletons is created once per jvm and shared across all FakeApplications. Here the singleton is `OptimizelyConfig`. If it is created only once then I can't test both the scenarios where the snippet is present and not. That is why I had to fork the jvm so that two tests can have different OptimizelyConfig.